parm/config/gfs/config.base

```diff
@@ -181,8 +182,11 @@ export DO_WAVE="NO"
 export DO_OCN="NO"
 export DO_ICE="NO"
 export DO_AERO="NO"
+export AERO_FCST_CDUMP="" # When to run aerosol forecast: gdas, gfs, or both
+export AERO_ANL_CDUMP="" # When to run aerosol analysis: gdas, gfs, or both
```

Comment:
Works for now to get it functional, but we should change the python so the previous method works correctly (and get rid of `WAVE_CDUMP` too).

Reply:
Noted.